common/core/gpudevice.cpp

```diff
@@ -824,19 +830,25 @@ void GpuDevice::InitializeFloatDisplay(
   }
 }
 
+void GpuDevice::InitGvtActive() {
+  int fd = GetFD();
+  int gvt_active = 0;
+  drm_intel_get_gvt_active(fd, &gvt_active);
+  if (gvt_active) {
+    gvt_active_ = true;
+    ITRACE("Running on GVT");
```

Comment:
suggest change to "Running on vGPU (GVT-g) mode"

Reply:
Updated